libraries/apollo-gradle-plugin/testProjects/language-version/build.gradle.kts

```diff
@@ -11,9 +11,9 @@ dependencies {
 }
 
 tasks.withType<KotlinCompile>().configureEach {
-  kotlinOptions {
+  compilerOptions {
     // Required for Kotlin < 1.6.10
     // See https://kotlinlang.org/docs/whatsnew1620.html#compatibility-changes-in-the-xjvm-default-modes
-    freeCompilerArgs += "-Xjvm-default=all"
+    freeCompilerArgs.add("-Xjvm-default=all")
```

Comment:
This doesn't need to change. The version we're testing with isn't bound to the version we're building with. I'll push a commit.